include/constants/battle_config.h

```diff
@@ -130,6 +130,10 @@
 #define B_PARALYZE_ELECTRIC         GEN_6 // In Gen6+, Electric type Pokémon can't be paralyzed.
 #define B_POWDER_GRASS              GEN_6 // In Gen6+, Grass type Pokémon are immune to powder and spore moves.
 
+// Critical Capture
+#define CRITICAL_CAPTURE            TRUE // if set to TRUE, critical capture will be enabled
+#define CATCHING_CHARM_BOOST        20   // % boost in critical capture odds if player has the catching charm
```

Comment:
follow the naming convention of `B_<NAME>`, so `B_CRITICAL_CAPTURE` and `B_CATCHING_CHARM_BOOST`